guides/training_keras_models_on_cloud.py

```diff
@@ -1,421 +1,244 @@
 """
 Title: Training Keras models with TensorFlow Cloud
-Author: [Jonah Kohn](https://jonahkohn.com)
+Authors: [Jonah Kohn](https://jonahkohn.com)[Sina Chavoshi](https://www.linkedin.com/in/sinachavoshi/)
 Date created: 2020/08/11
-Last modified: 2020/08/11
-Description: In-depth usage guide for TensorFlow Cloud.
+Last modified: 2021/07/23
+Description: Usage guide for TensorFlow Cloud.
 """
 """
 ## Introduction
 
-[TensorFlow Cloud](https://github.com/tensorflow/cloud) is a Python package that
-provides APIs for a seamless transition from local debugging to distributed training
-in Google Cloud. It simplifies the process of training TensorFlow models on the
-cloud into a single, simple function call, requiring minimal setup and no changes
-to your model. TensorFlow Cloud handles cloud-specific tasks such as creating VM
-instances and distribution strategies for your models automatically. This guide
-will demonstrate how to interface with Google Cloud through TensorFlow Cloud,
-and the wide range of functionality provided within TensorFlow Cloud. We'll start
-with the simplest use-case.
-"""
+<table class="tfo-notebook-buttons" align="left">
+  <td>
+    <a target="_blank" href="https://www.tensorflow.org/cloud/tutorials/overview.ipynb"><img src="https://www.tensorflow.org/images/tf_logo_32px.png" />View on TensorFlow.org</a>
+  </td>
+  <td>
+    <a target="_blank" href="https://colab.research.google.com/github/tensorflow/cloud/blob/master/g3doc/tutorials/overview.ipynb""><img src="https://www.tensorflow.org/images/colab_logo_32px.png" />Run in Google Colab</a>
+  </td>
+  <td>
+    <a target="_blank" href="https://github.com/tensorflow/cloud/blob/master/g3doc/tutorials/overview.ipynb"><img src="https://www.tensorflow.org/images/GitHub-Mark-32px.png" />View on GitHub</a>
+  </td>
+  <td>
+    <a href="https://storage.googleapis.com/tensorflow_docs/cloud/tutorials/overview.ipynb"><img src="https://www.tensorflow.org/images/download_logo_32px.png" />Download notebook</a>
+  </td>
+  <td>
+        <a href="https://kaggle.com/kernels/welcome?src=https://github.com/tensorflow/cloud/blob/master/g3doc/tutorials/overview.ipynb" target="blank">
+            <img width="90" src="https://www.kaggle.com/static/images/site-logo.png" alt="Kaggle logo">Run in Kaggle
+        </a>
+  </td>
+</table>
 
-"""
-## Setup
+TensorFlow Cloud is a library that makes it easier to do training and hyperparameter tuning of Keras models on Google Cloud.
 
-We'll get started by installing TensorFlow Cloud, and importing the packages we
-will need in this guide.
-"""
+Using TensorFlow Cloud's `run` API, you can send your model code directly to your Google Cloud account, and use Google Cloud compute resources without needing to login and interact with the Cloud UI (once you have set up your project in the console).
```

Comment:
Please keep lines under 80 characters (even text lines). This makes the file easier to review and more maintainable.

Reply:
Done.